clippy_utils/src/msrvs.rs

```diff
@@ -47,6 +47,7 @@ msrv_aliases! {
     1,18,0 { HASH_MAP_RETAIN, HASH_SET_RETAIN }
     1,17,0 { FIELD_INIT_SHORTHAND, STATIC_IN_CONST, EXPECT_ERR }
     1,16,0 { STR_REPEAT }
+    1,15,0 { COMBINED_MAYBE_BOUND }
```

Comment:
From 1.15.0, it supports `?Sized` in where clause, so can we change this name to `sized_in_where` or `maybe_bound_in_where` or etc.?
https://github.com/rust-lang/rust/pull/37791

Reply:
yep let's do that